website/src/content/api-reference/commercetools-frontend-application-components.mdx

```diff
@@ -597,6 +598,57 @@ const AccountPage = () => {
   />
   ```
 
+## TabHeader
+
+<Playground urlPath="tabular-modal-page" />
```

Comment:
I added the same visual as in the `TabularModalPage` section since it's hard to showcase it on it's own